### PR TITLE
Adds numeric keyCode example to keyIsDown docs

### LINKED ISSUE #3036: Suggested improvement to keyIsDown documentation
I had a suggestion for improving this page:

https://p5js.org/reference/#/p5/keyIsDown

The only examples are things like keyIsDown(UP_ARROW) and there aren't any examples of just putting a keyCode in the keyIsDown function. In the description it is clear that this should work, but I totally missed it the first 20 times I read it and I assumed that UP_ARROW, DOWN_ARROW, RIGHT_ARROW and LEFT_ARROW were the only things that keyIsDown worked with.

An improvement would be to add another if statement to the example program like this:
```js
if ( keyIsDown(65) ) {   // 65 is the keyCode for the "a" key
     x -= 5;
}
```
I think this would clear this up.

### PR DESCRIPTION
**Pull request by bomoko**
Fixes #3036

### CODE REVIEW

File: src/events/keyboard.js [hunk 1]
```diff
@@ -322,6 +322,7 @@ p5.prototype._onblur = function(e) {
  * <div><code>
  * var x = 100;
  * var y = 100;
+ * var widthAndHeight = 50;
```
Comment (lmccart) [inline src/events/keyboard.js]: could you rename this "diameter" to be a little less clunky?

**Comment by bomoko** [comment]
No problemo @lmccart! Sorry about the duh moment on the diameter :)
Hope that all looks good now.

**Comment by lmccart** [comment]
thanks looks great. haha ohh i can relate.. ;)

**Comment by kjhollen** [comment]
thanks @bomoko !

**Pull request merged**